engine/src/CalciteInterpreter.cpp

```diff
@@ -31,14 +32,22 @@ std::shared_ptr<ral::cache::graph> generate_graph(std::vector<ral::io::data_load
 		auto max_kernel_id = std::get<1>(query_graph_and_max_kernel_id);
 		auto output = std::shared_ptr<ral::cache::kernel>(new ral::batch::OutputKernel(max_kernel_id, queryContext.clone()));
 
-		if(logger){
-            logger->info("{query_id}|{step}|{substep}|{info}|||||",
-                                        "query_id"_a=queryContext.getContextToken(),
-                                        "step"_a=queryContext.getQueryStep(),
-                                        "substep"_a=queryContext.getQuerySubstep(),
-                                        "info"_a="\"Query Start\n{}\""_format(tree->to_string()));
+		if(batchLogger){
+            batchLogger->info("{query_id}|{step}|{substep}|{info}|||||",
+                              "query_id"_a=queryContext.getContextToken(),
+                              "step"_a=queryContext.getQueryStep(),
+                              "substep"_a=queryContext.getQuerySubstep(),
+                              "info"_a="\"Query Start\n{}\""_format(tree->to_string()));
 		}
 
+        if(queriesLogger){
+            queriesLogger->info("{query_id}|{step}|{substep}|{info}|||||",
```

Comment:
The format of the queriesLogger is a little different. Its something like this:

        logger->info("{ral_id}|{query_id}|{start_time}|{plan}|{sql}",
                                        "ral_id"_a=queryContext.getNodeIndex(communicationData.getSelfNode()),
                                        "query_id"_a=queryContext.getContextToken(),
                                        "start_time"_a=eventTimer.start_time(),
                                        "plan"_a=query,
                                        "sql"_a=tree->to_string());